exasol/toolbox/templates/github/workflows/merge-gate.yml

```diff
@@ -0,0 +1,46 @@
+name: Merge-Gate
+
+on:
+  workflow_call:
+    secrets:
+      ALTERNATIVE_GITHUB_TOKEN:
+        required: false
+
+jobs:
+
+  fast-checks:
+    name: Fast
+    uses: ./.github/workflows/checks.yml
+
+  slow-checks:
+    name: Slow
+    runs-on: ubuntu-latest
+
+    # Even though the "manual-approval" environment will be created automatically,
+    # it still needs to be configured to require manual approval.
+    # See project settings on GitHub (Settings / Environments / manual-approval).
+    environment: manual-approval
+
+    # Replace the steps below with the required actions 
```

Comment:
Could you please copy the latest version of the PTB workflow to the template?

Reply:
Good catch, thx :+1: . Fixed